EosPayload/drivers/telemetry_I2C_driver.py

```diff
@@ -61,11 +64,18 @@ def device_read(self, logger: logging.Logger) -> None:
                 logger.warning(f"exception occurred while logging data: {e}\n{traceback.format_exc()}")
 
             current_time = datetime.now()
+            pressure = self.mpr.pressure
+            humidity = 1
```

Comment:
why are you setting humidity to 1?

Reply:
The MPRLS pressure sensor only has pressure. we don't have humidity

Reply:
i can see that...but why 1 and not something less arbitrary like 0?

Reply:
because according to EosLib logic, it is expecting a value between 0 and 100. I tried -1 and it did not work

Reply:
I can try 0

Reply:
0 is valid

Reply:
fixed